### PR TITLE
LPS-154857 rename properties: module.framework.properties.dependency.* , module.framework.properties.initial.system.check.enabled

### PR DESCRIPTION
**Pull request by W2jx**
Hi @dantewang, I have used the new method in code.

### CODE REVIEW

File: modules/apps/portal-osgi-debug/portal-osgi-debug-impl/src/main/java/com/liferay/portal/osgi/debug/internal/osgi/commands/SystemCheckOSGiCommands.java [hunk 1]
```diff
@@ -61,10 +63,24 @@ protected void activate(BundleContext bundleContext) {
...
 		}
 
-		if (GetterUtil.getBoolean(
-				bundleContext.getProperty("initial.system.check.enabled"),
-				true)) {
+		boolean checkEnabled = true;
+		ServiceReference<Props> serviceReference =
+			bundleContext.getServiceReference(Props.class);
 
+		if (serviceReference != null) {
```
Comment (dantewang) [inline modules/apps/portal-osgi-debug/portal-osgi-debug-impl/src/main/java/com/liferay/portal/osgi/debug/internal/osgi/commands/SystemCheckOSGiCommands.java:70]: @W2jx `SystemCheckOSGiCommands` is a `@Component` so you can just `@Reference Props _props` and use it.

File: modules/apps/static/portal-dependency-manager-component-executor-factory/portal-dependency-manager-component-executor-factory/src/main/java/com/liferay/portal/dependency/manager/component/executor/factory/internal/activator/ComponentExecutorFactoryBundleActivator.java [hunk 2]
```diff
@@ -40,17 +43,29 @@
...
-		boolean threadPoolEnabled = GetterUtil.getBoolean(
-			bundleContext.getProperty("dependency.manager.thread.pool.enabled"),
-			true);
+		boolean threadPoolEnabled = true;
+
+		long syncTimeout = 60;
+
+		_propsServiceReference = bundleContext.getServiceReference(Props.class);
+
+		if (_propsServiceReference != null) {
```
Comment (dantewang) [inline modules/apps/static/portal-dependency-manager-component-executor-factory/portal-dependency-manager-component-executor-factory/src/main/java/com/liferay/portal/dependency/manager/component/executor/factory/internal/activator/ComponentExecutorFactoryBundleActivator.java:52]: @W2jx We know for sure that there is already a `Props` OSGi service at this point. If it doesn't exist it means bigger problem in our system. So I don't think the null protections here are needed. Or you can just use portal-kernel's `PropsUtil`.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
ci:stop

**Comment by W2jx** [comment]
ci:stop

**Pull request closed without merging**